File: BookingApp/frontend/src/app/shared/models/DeleteAccountRequestReplyDTO.ts
export class DeleteAccountRequestReplyDTO{
    public deleteAccountRequestId : Number;
    public description : String;
    public clientId : Number;

    constructor(deleteAccountRequestId : Number,description : String,clientId : Number ){
        this.deleteAccountRequestId = deleteAccountRequestId;
        this.description = description;
        this.clientId = clientId;
    }

}

File: BookingApp/src/main/java/com/example/BookingApp/users/dto/DeleteAccountRequestReplyDTO.java
package com.example.BookingApp.users.dto;

import lombok.*;
import org.springframework.beans.factory.annotation.Required;
import org.springframework.web.bind.annotation.RequestMapping;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class DeleteAccountRequestReplyDTO {
    private Long deleteAccountRequestId;
    private String description;
    private Long clientId;
}
